antelope_contracts/tests/erc20/erc20_tester.cpp

```diff
@@ -78,7 +78,7 @@ erc20_tester::erc20_tester(std::string native_symbol_str) : native_symbol(symbol
                 token_account,
                 mvo()("to", faucet_account_name)("quantity", asset(1'000'000'000'0000, symbol::from_string("4,USDT")))("memo", ""));
 
-    set_code(evmin_account, testing::contracts::evm_depoit_proxy_wasm());
+    set_code(evmin_account, testing::contracts::evm_deposit_proxy_wasm());
```

Comment:
Also call `set_abi`.

Reply:
If we include the `set_abi` call, it will raise an exception since the abi file is not there